### DISCUSSION TITLE
Reset password code

### DISCUSSION THREAD

**Discussion by nafiulhaq** [community/community#129659] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am trying to add feature to my website like reset password code how to do that

**Comment by nafiulhaqueinan** [comment]
you can do it useing node.js packages need for this :
express nodemailer crypto jsonwebtoken body-parser

**Comment by nafiulhaqueinan** [comment]
another you can use  bcrypt do  google for them
